internal/config/logging.go

```diff
@@ -30,4 +30,5 @@ func (cfg *logging) parseConfigValues() error {
 func (cfg logging) loadDefaultValues(v *viper.Viper) {
 	v.SetDefault("log.structured", false)
 	v.SetDefault("log.file", "")
+	v.SetDefault("log.level", string(logger.WarnLevel))
```

Comment:
Nice! Was about to ask about the todo that went away.

Reply:
yeah, it felt quite nice to get rid of the multiple-fields to represent a logger level in the app config. And since the upstream has this field as essentially a string, we can use it directly like this 🎉 .